config/routes.rb

```diff
@@ -92,7 +92,7 @@
     member do
       patch :activate
       patch :deactivate
-      patch :resend_invitation
+      get :resend_invitation
```

Comment:
I am very ambivalent about a modifying operation being a get. I would prefer patch or post. Why does this need to be a get? Does it have to do with calling it from an email?

Reply:
Exactly, can call a patch from email.

Reply:
Gotcha gotcha
Maybe someday we will find a better way, but at least it's authenticated 😅

Reply:
Exactly.